looptypes/interface.go

```diff
@@ -1,4 +1,4 @@
-package loop
+package looptypes
 
 import (
 	"time"
```

Comment:
It looks like there is more moved to the new package than necessary. For example `OutRequest`.

Reply:
So you would suggest to only move the types into this new module that are currently used by the server? This feels a bit strange to me to have these core types in multiple packages.